java/test/jmri/jmrix/pi/RaspberryPiSensorManagerTest.java

```diff
@@ -102,6 +103,11 @@ public void testPullResistanceConfigurable(){
        Assert.assertTrue("Pull Resistance Configurable",l.isPullResistanceConfigurable());
     }
 
+    @Override
+    @Ignore // the parent test doesn't work right for these names
```

Comment:
The actual error generated here from the tests:

ERROR - Provisioning sensor PIS9 failed with: This GPIO pin already exists: GPIO 9 [main] jmri.jmrix.pi.RaspberryPiSensor.init()

doesn't have to do with the name itself.  The issue is that the Pi4J library which is used for our RaspberryPi support uses some static structures that aren't fully reset between tests.

Our PiGpioProviderScaffold hides the parts of the Pi4J library that prevent this from running on anything but a RaspberryPi, but it doesn't do anything with the pin provisioning parts of the library, which is what ultimately generates the complaint.

The problem is there is another test in this test file that uses pin 9 (which happens to be the test you just added). 

I'll fix this once it is merged.

Reply:
@pabender Thank you! I had misunderstood and thought the superclass was generating bad systemnames. Thanks for catching that.